src/ucar/unidata/idv/chooser/catalogref.xml

```diff
@@ -1,24 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE catalog SYSTEM "http://www.unidata.ucar.edu/projects/THREDDS/xml/InvCatalog.0.6.dtd">
-<catalog name="Example THREDDS Inventory Catalog extra really good version" version="0.6" xmlns="http://www.unidata.ucar.edu/thredds" xmlns:xlink="http://www.w3.org/1999/xlink">
+<!DOCTYPE catalog SYSTEM "https://www.unidata.ucar.edu/projects/THREDDS/xml/InvCatalog.0.6.dtd">
+<catalog name="Example THREDDS Inventory Catalog extra really good version" version="0.6" xmlns="https://www.unidata.ucar.edu/thredds" xmlns:xlink="https://www.w3.org/1999/xlink">
 	<dataset name="Example THREDDS Inventory Catalog Final extra good version">
 		<!--	all referenced servers -->
-		<service name="ACD" serviceType="DODS" base="http://www.acd.ucar.edu/dods/testServer/"/>
+		<service name="ACD" serviceType="DODS" base="https://www.acd.ucar.edu/dods/testServer/"/>
 		<service name="local" serviceType="NetCDF" base="file://E:/metapps/data/"/>
 		<service name="Motherlode" serviceType="Compound" base="">
 		        <service name="MotherlodeFTP" serviceType="FTP" base="ftp://motherlode.ucar.edu/ftp/pub/thredds/"/>
-			<service name="MotherlodeDODS" serviceType="DODS" base="http://motherlode.ucar.edu/cgi-bin/dods/"/>
+			<service name="MotherlodeDODS" serviceType="DODS" base="https://motherlode.ucar.edu/cgi-bin/dods/"/>
 		</service>
 		<!-- simple collection -->
                 <action name="Radar chooser" url="/ucar/unidata/idv/chooser/radarchooser.xml"/>
                 <action name="Catalog4" url="/ucar/unidata/idv/chooser/catalog4.xml"/>
                 <action name="Contained test" >
                    <datasetChoice name="NEXRAD Level 3 Radar"
-                       actiontemplate="http://motherlode.ucar.edu/cgi-bin/thredds/RadarServer.pl?stn=FTG&amp;product=N0R&amp;time=1hour">

```

Comment:
Any motherlode.ucar.edu URLs are probably no longer available. I think motherlode redirects to atm.ucar.edu but I'm pretty sure there hasn't been any cgi-bin content in quite some time.